src/console_implementation/main_header.hpp

```diff
@@ -5,16 +5,21 @@
 #include <memory>
 #include <thread>
 #include <chrono>
+#include <string>
+#include <fstream>
+#include <sstream>
 
 //Library definitions
 #include "../business_logic/include/Tplayer.hpp"
 #include "Idrawable.hpp"
 #include "drawable_player.hpp"
+#include "../business_logic/include/Tlevel.hpp"
 
 //Global game loop functions
 
 bool input_processing(const std::shared_ptr<Ientity>& player);
-WINDOW* initial_window_gameplay_output();
+WINDOW* initial_window_gameplay_output(std::shared_ptr<Ilevel> p_current_level, unsigned int height, unsigned int width);
 WINDOW* initial_window_scoreboard_output();
```

Comment:
Consider not returning raw pointer. Better practice is to return smart pointer - RAII.
```suggestion
std::shared_ptr<WINDOW> initial_window_gameplay_output(std::shared_ptr<Ilevel> p_current_level, unsigned int height, unsigned int width);
std::shared_ptr<WINDOW> initial_window_scoreboard_output();
```